apps/ledger-live-mobile/src/components/ValidateOnDevice.tsx

```diff
@@ -181,15 +183,20 @@ export default function ValidateOnDevice({
       ? transTitleWording
       : t("ValidateOnDevice.title.send", getDeviceModel(device.modelId));
 
+  const isBigLottie = device.modelId === DeviceModelId.nanoFTS;
+
   return (
     <RootContainer>
-      <ScrollContainer>
+      <ScrollView
+        // @ts-expect-error this property actually working and the only way to do what we want (contentContainerStyle doesn't work)
```

Comment:
I don't know how this could do anything !
You can put it in the style prop
Or use the styled method from ui lib on ScrollView so you can use this prop

Reply:
in the style prop it crashes

Reply:
same with `styled`